### ISSUE TITLE
tzinfo vs. timestamp type for timezone.utc

### ISSUE THREAD

**Issue by dlinnemeyer** [python/typeshed#567]
I was running mypy on an existing project, and the following code (recommended by the python datetime docs) ran into an error in mypy, which seems to stem from typeshed.

```python
datetime.fromtimestamp(timestamp, timezone.utc)
```

Error from mypy is `Argument 2 to "fromtimestamp" of "datetime" has incompatible type "tzinfo"; expected "timezone"`

It looks like fromtimestamp expects an instance of timezone, but timezone.utc is stubbed as a tzinfo class, which looks like a parent of timezone?

Relevant sections in /stdlib/3/datetime.pyi are lines 16-19

```python
class timezone(tzinfo):
    utc = ...  # type: tzinfo
    min = ...  # type: tzinfo
    max = ...  # type: tzinfo
```

and 175-ish, under class datetime:

```python
    @classmethod
    def fromtimestamp(cls, t: float, tz: timezone = ...) -> datetime: ...
```

**Comment by gvanrossum** [comment]
Hm, you're right, it seems this needs a small adjustment to the stub. Do
you feel capable of submitting a PR with a fix? (Also, does the Python 2
stub has the same issue perhaps?)

**Comment by dlinnemeyer** [comment]
Sure, I'll take a shot at it.

**Comment by dlinnemeyer** [comment]
Okay, that's up.

**Comment by gvanrossum** [comment]
Closed by #568. (Note: If you include "Fixes #568" in the body of the commit message it will auto-close the issue.)

**Comment by dlinnemeyer** [comment]
Got it. Thanks.
